bin/yaml/c.yaml

```diff
@@ -50,6 +50,35 @@ compilers:
       check_stderr_on_stdout: true
       targets:
         - '1.27'
+    dpu-upmem-dpurte-clang:
+      type: tarballs
+      compression: gz
+      subdir: upmem-dpu
+      dir: "{subdir}/upmem-{name}-Linux-x86_64"
+      untar_dir: "upmem-{name}-Linux-x86_64"
+      check_exe: bin/clang --version
+      url: http://sdk-releases.upmem.com/{name}/ubuntu_22.04/upmem-{name}-Linux-x86_64.tar.gz
+      targets:
+        # - 2019.1.0
```

Comment:
What are these commented-out lines? can we remove them unless we plan to install them, in which case can we comment them back in? thanks!

Reply:
Sorry I didn't spot you mentioned this in the PR itself...apologies. Can you please remove? :) thanks!

Reply:
I removed them, I suppose its not that important host earlier versions of this compiler since the entire architecture is quite new in the first place